.github/workflows/auto-pr-submodules.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Submodules and Submit a PR
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    schedule:
+      - cron:  '0 0 * * *'
+
+jobs:
+  createPullRequest:
+    if: contains(github.ref, 'master')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Update Submodules to Remote Latest
+        run: python3 .github/scripts/google-skywater-pdk-roll.py -l libraries
+
+      - name: Export Body
+        run: echo "BODY=$(git log -1 --pretty=%B)" >> $GITHUB_ENV
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ github.token }}
+          title: 'Update Submodules to Remote Latest'
```

Comment:
Maybe set title through another envvar, similarly to BODY?

Reply:
@umarcor: It's possible, but what determines the title?

Reply:
Anything that allows telling them apart. If multiple PRs are to be created, all of them having the same title makes it difficult to search later. For instance, you could add the short sha of the commit that triggered the PR.

Reply:
@umarcor: If you mean simultaneous PRs, then this will never happen because the branch will rebase itself to master whenever a push happens there as long as the PR is still open. But, if you mean historically, then the commit SHA (although easy to add to the title) wouldn't be easy to look up either. Still, I added this anyway, since if it's there, we can modify it later.